catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -67,65 +70,64 @@ public Metacard transform(NitfSegmentsFlow nitfSegmentsFlow, Metacard metacard)
 
         nitfSegmentsFlow.fileHeader(header -> handleHeader(header, metacard));
 
-        String locationString = setLocation(metacard);
+        transformTargetLocation(metacard);
+        transformAircraftLocation(metacard);
+
+        return metacard;
+    }
+
+    private void handleHeader(TaggedRecordExtensionHandler header, Metacard metacard) {
+        List<Tre> tres = header.getTREsRawStructure().getTREs();
+
+        tres.stream().filter(tre -> ACFTB.equals(tre.getName().trim()))
+                .forEach(tre -> handleSegmentHeader(metacard, tre, AcftbAttribute.values()));
+
+        tres.stream().filter(tre -> MTIRPB.equals(tre.getName().trim())).forEach(tre -> {
+            handleSegmentHeader(metacard, tre, MtirpbAttribute.values());
+
+            try {
+                List<TreGroup> targets = tre.getEntry(TARGETS).getGroups();
+
+                targets.stream().forEach(group -> handleSegmentHeader(metacard, group,
+                        IndexedMtirpbAttribute.values()));
+            } catch (NitfFormatException e) {
+                LOGGER.error(e.getMessage(), e);
```

Comment:
probably can log this at WARN.  Also you can replace e.getMessage() with something more admin friendly as the exception message will get logged with the exception.